README.md

```diff
@@ -1,8 +1,8 @@
-# eslint-plugin-markdown
+# ESLint Markdown Language Plugins
```

Comment:
"Plugins" sounds like there will be multiple plugins here, but we'll have only one?

Reply:
That's just a typo. :)